src/client/datascience/kernel-launcher/kernelEnvVarsService.ts

```diff
@@ -51,7 +51,7 @@ export class KernelEnvironmentVariablesService {
 
         let [customEditVars, interpreterEnv] = await Promise.all([
             this.customEnvVars.getCustomEnvironmentVariables(resource).catch(noop),
-            interpreter && interpreter.envType == EnvironmentType.Conda
```

Comment:
This was wrong, we should get activated env variables for Virtual Env, etc, not just Conda environments.

Reply:
This will work for virtual environments